### PR TITLE
[p-s] Fix @ActionOutput annotations

### PR DESCRIPTION
**Pull request by lolodomo**
Related to #17636

### CODE REVIEW

File: bundles/org.openhab.binding.solarforecast/src/main/java/org/openhab/binding/solarforecast/internal/actions/SolarForecastActions.java [hunk 1]
```diff
@@ -48,7 +48,7 @@ public class SolarForecastActions implements ThingActions {
     private Optional<ThingHandler> thingHandler = Optional.empty();
 
     @RuleAction(label = "@text/actionDayLabel", description = "@text/actionDayDesc")
-    public @ActionOutput(name = "getEnergyOfDay", type = "QuantityType<Energy>") QuantityType<Energy> getEnergyOfDay(
```
Comment (lsiepel) [inline bundles/org.openhab.binding.solarforecast/src/main/java/org/openhab/binding/solarforecast/internal/actions/SolarForecastActions.java:51]: I i understand it correct; when omitted the name defaults to 'succes'. Is that the right value for this method?

Comment (lolodomo) [reply bundles/org.openhab.binding.solarforecast/src/main/java/org/openhab/binding/solarforecast/internal/actions/SolarForecastActions.java:51]: The name defaults to "result". This method has a unique return value being a `QuantityType<Energy>`.

Each time there is a unique return value, we can omit the "name" parameter but we must set a proper "label" parameter that will be displayed.

Comment (lsiepel) [reply bundles/org.openhab.binding.solarforecast/src/main/java/org/openhab/binding/solarforecast/internal/actions/SolarForecastActions.java:51]: Thanks for the clarification.

**Review by lsiepel — APPROVED** [review]
Thanks, LGTM

**Review by druciak — COMMENTED** [review]
LGTM

**Pull request merged**